docs/src/get-started/deploy/init.md

```diff
@@ -24,6 +24,7 @@ Then go through each of the steps to create the project:
    The CLI lists each region's location, cloud provider, and average carbon intensity.
 3. Choose a plan.
    A Development plan is enough before you deploy anything.
+   You can't choose production plans during a free trial.
```

Comment:
Should it be mentioned that, as a consequence, a free trial user won't have to choose a plan?

Reply:
That was the intent. I'll try to improve it.

Reply:
```suggestion
   During a free trial, you can't choose production plans so you won't see this option.
```